File: leetcode/1-two-sum/two-sum.cs
public class Solution
{
    public int[] TwoSum(int[] nums, int target)
    {
        int[] indices = new int[2];
        
        for (int i = 0; i < nums.Length; i++)
        {
            for (int j = 0; j <  nums.Length; j++)
            {
                if (i == j)
                {
                    continue;
                }
                else if (nums[i] + nums[j] == target)
                {
                    indices[0] = i;
                    indices[1] = j;
                }
            }
        }

        return indices;
    }
}

File: leetcode/1-two-sum/two-sum.ts
function twoSum(nums: number[], target: number): number[] {
    for (let i:number = 0; i < nums.length; i++) {
        for (let j:number = 0; j < nums.length; j++) {
            if (i === j) { continue; }

            if (nums[i] + nums[j] === target) {
                return [i, j];
            }
        }
    }
    return []; // <-- this is ugly but avoids VS Code error
};